src/JsonApi/Serializer/ErrorNormalizer.php

```diff
@@ -52,6 +52,10 @@ public function normalize($object, $format = null, array $context = [])
             $data['trace'] = $trace;
         }
 
+        if (0 !== $code = $object->getCode()) {
+            $data['code'] = $code;
```

Comment:
According to the JSON:API spec, the code must be casted as a string.

Reply:
this is outdated and no longer needed as the ExceptionInterface makes sure getCode returns a string.